test/core_tests.js

```diff
@@ -0,0 +1,138 @@
+QUnit.begin(function( details ) {
+  // In case tests didn't complete, we are left with a hash/anchor pointing somewhere. But we want to start from scratch:
+  window.location.hash = "";    
+});
+
+QUnit.test( "Initialize Impress.js", function( assert ) {
+  assert.ok( impress, 
+             "impress declared in global scope" );
+  assert.strictEqual( impress().init(), undefined,
+                    "impress().init() called." );
+  assert.strictEqual( impress().init(), undefined,
```

Comment:
What is the value on testing multiple calls of impress.js API if we are not objectively determining if the behavior was actually noop or not (like checking some global variable or something like that?). This seems to be just a smoke test to ensure that multiple impress.js calls don't trigger an error, is my assumption correct?